# HG changeset patch
# User Rik <rik@octave.org>
# Date 1428772583 25200
#      Sat Apr 11 10:16:23 2015 -0700
# Node ID 3b90259f17f32083f44d97a191512735cc0c9106
# Parent  8e42898f4d731f89e1e46d5f87462349c9515db0
Document that ginput() is intended for 2-D plots (bug #44427).

* ginput.m: Update docstring with implementation note about ginput
being used for 2-D plots.

diff --git a/scripts/plot/util/ginput.m b/scripts/plot/util/ginput.m
--- a/scripts/plot/util/ginput.m
+++ b/scripts/plot/util/ginput.m
@@ -25,16 +25,20 @@
 ## If @var{n} is defined, then capture @var{n} events before returning.
 ## When @var{n} is not defined @code{ginput} will loop until the return key
 ## @key{RET} is pressed.
 ##
 ## The return values @var{x}, @var{y} are the coordinates where the mouse
 ## was clicked in the units of the current axes.  The return value @var{button}
 ## is 1, 2, or 3 for the left, middle, or right button.  If a key is pressed
 ## the ASCII value is returned in @var{button}.
+##
+## Implementation Note: @code{ginput} is intenteded for 2-D plots.  For 3-D
+## plots see the @var{currentpoint} property of the current axes which can be
+## transformed with knowledge of the current @code{view} into data units.
 ## @seealso{gtext, waitforbuttonpress}
 ## @end deftypefn
 
 function varargout = ginput (n = -1)
 
   if (nargin > 1)
     print_usage ();
   endif
